### ISSUE TITLE
Refactor repeated codes in Api

### ISSUE THREAD

**Issue by reyraa** [LiskHQ/lisk-explorer#281]
### Expected behaviour
The Api must follow DRY programming Principle

### Actual behaviour
Many code parts i Api are not repetitive and can be simplified by following DRY principle. Try creating common modules and removing repetitive codes

**Comment by arefm** [comment]
Ok, it a valid point I didn't think it though. A solution would be instead of registering route in every file. we define route configs as arrays. something like this:

routes.js:
```javascript

module.exports = [{
	path: '/getLastBlocks',
	handler: 'getLastBlocks',
	service: 'blocks',
	data: () => { /* pass required data */ }
}, {
	path: '/getBlock',
	handler: 'getBlock',
	service: 'blocks',
	data: () => { /* pass required data */ }
}, {
	path: '/getHeight',
	handler: 'getHeight',
	service: 'blocks',
	data: () => { /* pass required data */ }
}, {
	path: '/getBlockStatus',
	handler: 'getBlockStatus',
	service: 'blocks',
	data: () => { /* pass required data */ }
}]

```

then import those into index.js and register them dynamically:

```javascript

const routes = require('./routes.js')

routes.forEach(route => {
	serviceModule = require(`../lib/api/${route.service}`)
	service = new serviceModule(app)
	app.get(`/api/${route.path}`, (req, res, next) => {
		handler(service, route.handler, [PASS DATA HERE], req, res, next)
	})
})

```

**Comment by reyraa** [comment]
Closed by #282
